.github/workflows/tests.yml

```diff
@@ -173,11 +173,11 @@ jobs:
 
       - name: Core RSpec
         if: matrix.build_type == 'backend' && matrix.target == 'core'
-        run: bin/turbo_rspec --verbose
+        run: bin/turbo_rspec --use-runtime-info --verbose
```

Comment:
Note that we no files are passed as arguments, runtime info is enabled by default but I'm adding the `--use-runtime-info` flag for clarity sake.